index.d.ts

```diff
@@ -76,6 +81,7 @@ declare module 'replicate' {
       options: {
         input: object;
         wait?: boolean | { interval?: number; maxAttempts?: number };
```

Comment:
```suggestion
        wait?: false | { interval?: number; maxAttempts?: number };
```

Reply:
Not sure I understand the suggestion here. (I am not good at TypeScript). Does specifying false indicate that the type is boolean and the default value is false?